language/compiler/ir-to-bytecode/src/compiler.rs

```diff
@@ -475,12 +476,14 @@ pub fn compile_script<'a>(
         type_parameters: sig.type_parameters,
         parameters: parameters_sig_idx,
         code,
+    };
+    match BoundsChecker::verify_script(&script) {
```

Comment:
Seems sane to keep this here, since the IR is not supposed to be able to produce modules that don't bounds check